sql/testdata/rename

```diff
@@ -0,0 +1,74 @@
+query T
```

Comment:
can you add a test either here, or in privilege_database to check that only root can rename a DB, even if the user has WRITE on the db